### PR TITLE
Bug fixes for testing errors

### PR DESCRIPTION
**Pull request by RyanMersmann**
# Description

Fixes #

- What work was done? changed variable props.child.VotesRemaining -> props.VoteRemaining
- Why was this work done? to get the tests to pass in order to get updates to merge
- What feature / user story is it for? as a user, I want a bug free experience

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [y ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested
- [y ] Yes

## Checklist

- [y] My code follows the style guidelines of this project
- [y] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by cearc-sara — CHANGES_REQUESTED** [review]
everything looks great except the one change and then we are good to merge! Great job everyone for fixing these tests!

**Review by cearc-sara — APPROVED** [review]
awesome looks great

File: src/components/pages/MatchUp/MatchUpContainer.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default connect(
     child: state.child,
     faceoffs: state.faceoffs,
     votes: state.votes,
-    custom_date: state.date.custom_date,
+    custom_date: state.date,
```
[the remark this answers is not in the document]
Comment (RyanMersmann) [reply src/components/pages/MatchUp/MatchUpContainer.js]: made that change, thanks!

**Review by leachtucker — APPROVED** [review]
Looks great!

**Pull request merged**